composer.json

```diff
@@ -4,12 +4,12 @@
     "license": "MIT",
     "description": "Rector upgrades rules for Symfony Framework",
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.0",
         "ext-xml": "*",
-        "rector/rector": "^0.11",
         "danielstjules/stringy": "^3.1"
     },
     "require-dev": {
+        "rector/rector-src": "^0.11",
         "phpunit/phpunit": "^9.5",
```

Comment:
This avoids using `rector/rector` duplicated classes from Rector\Symfony namespace